verification.py

```diff
@@ -363,6 +363,58 @@ def f():
             self._run_mutation_discoal_stats(key, command_line)
         self._instances[key] = f
 
+    def run_sweep_comparison(self, key, args):
+        """
+        does a comparison of sweep model vs discoal
+        """
+        df = pd.DataFrame()
+        refsize = 1e4
+        nreps = 1000
+        mod = msprime.SweepGenicSelection(refsize)
+        mod.start_frequency = 1.0 / (2 * refsize)
+        mod.end_frequency = 1.0 - (1.0 / (2 * refsize))
+        mod.alpha = 1000.0
+        mod.dt = 1.0/(40 * refsize)
+        seqlen = 1e4
+        mu = 2.5e-4
+        rho = 2.5e-4
+        mod.position = np.floor(seqlen/2)
+        data = collections.defaultdict(list)
+        replicates = msprime.simulate(10,  model=mod, length=seqlen,
+                                      num_labels=2, recombination_rate=rho,
+                                      num_replicates=nreps)
+        for x in replicates:
+            ts = msprime.simulate(from_ts=x, recombination_rate=rho)
+            tsm = msprime.mutate(ts, rate=mu)
+            data["pi"].append(tsm.diversity(span_normalise=False))
+            data["D"].append(tsm.Tajimas_D())
+            data["ss"].append(tsm.segregating_sites(span_normalise=False))
+        data["pi"] = np.array(data["pi"]).flatten()
+        data["D"] = np.array(data["D"]).flatten()
+        data["ss"] = np.array(data["ss"]).flatten()
+        df = pd.DataFrame.from_dict(data)
+        df = df.fillna(0)
+        df_d = self._run_discoal_mutation_stats(args)
+        df_df = df_d[["pi", "D", "ss"]]
+        print("msp pi mean: ", df['pi'].mean())
```

Comment:
this is just sanity checking stuff that is helping me debug scaling. I imagine I'll take it out before we merge